docs/kola/external-tests.md

```diff
@@ -103,6 +103,52 @@ it out.
 (Previously the API for this was to send `SIGTERM` to the current process; that
 method is deprecated and will be removed at some point)
 
+## HTTP Server
+
+The `kolet` binary is copied into the CoreOS system running the tests. Notably,
+it contains the built-in command `kolet httpd [port] [path]` for starting an
+HTTP file server at port to serve the contents of the file system rooted at path.
+
+To have an HTTP server started before your tests runs, you can create an Ignition
+config (or Fedora CoreOS config) and include it in your external test directory.
+
+In the following Fedora CoreOS config example, the Ignition config includes a
+path unit and a service unit. The path unit ensures that the httpd service runs
+automatically once the Kolet binary is copied to the system. An HTTP server will
+be started at `localhost:8080` and serve the files in `/var/home/core`. Your test
+can then do e.g. `curl localhost:8080/hello_world.txt`.
+
+Example:
+```
+variant: fcos
+version: 1.1.0
+systemd:
+  units:
+    - name: kolet-httpd.path
+      enabled: true
+      contents: |
+        [Path]
+        PathExists=/var/home/core/kolet
+        [Install]
+        WantedBy=multi-user.target
```

Comment:
In particular, to avoid race conditions this needs to be `WantedBy=kola-runext.service` (which isn't documented right now but should be).